style.css

```diff
@@ -50,3 +50,4 @@ ul{
 h1{
     font-size: 1rem;
 }
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,500,700');
```

Comment:
Sorry, I don't think that we need another font. In this section the original website usees only font "Lato" which is already imported in the index.html `<head>` section line 7.